plugin/build.gradle.kts

```diff
@@ -53,6 +54,43 @@ pluginBundle {
     }
 }
 
+configure<PublishingExtension> {
```

Comment:
You can remove this `PublishingExtension` completely as it's not needed.

Reply:
Not sure I follow. I could replace `configure<PublishingExtension> {}` with `publishing {}` to leverage generated accessors. I've been avoiding this lately though as I had the impression that intelliJ performed better without the generated accessors. 
Happy to change to `publishing {}` though as I don't really have any data to back this impression.

Reply:
Ah, now I got it! The `publication` block isn't required as the `com.gradle.plugin-publish` plugin already creates a `pluginMaven` publication. I'll remove it.

I kept the `repositories` block still as I feel using `mavenLocal` could be a bit dangerous (see above).

Reply:
> could be a bit dangerous (see above).

How do you feel about https://github.com/Yelp/swagger-gradle-codegen/pull/87#discussion_r374580723 ?